Dockerfile

```diff
@@ -1,37 +1,39 @@
-FROM phusion/baseimage:0.9.19
+FROM phusion/baseimage:0.9.22
```

Comment:
Good question - I stuck with phusion because it was the one that was already being used by this Dockerfile (which I did not write), but I think I have talked to other programmers in the lab and the consensus is that there was really no super-good reason for using it. Which other Ubuntu 16.04 images do you recommend?